src/main/xerus/monstercat/api/Player.kt

```diff
@@ -158,10 +167,30 @@ object Player: FadingHBox(true, targetHeight = 25) {
 	}
 	
 	private val pauseButton = ToggleButton().id("play-pause").onClick { if(isSelected) player?.pause() else player?.play() }
-	private val stopButton = buttonWithId("stop") { reset() }
+			.apply {
```

Comment:
please reformat all these lines properly

Reply:
Fixed in 69972f7dac09193feff88be69eda614e4011e492

Reply:
Tell me if this is good : e2694bfdb821291d86cd20f13dfb21c38d383214

Reply:
Is this ok ? e2694bfdb821291d86cd20f13dfb21c38d383214